package-lock.json

```diff
@@ -6885,7 +6885,7 @@
       "requires": {
         "@babel/code-frame": "^7.0.0",
         "jest-worker": "^23.2.0",
-        "serialize-javascript": "^2.1.2",
+        "serialize-javascript": "^1.5.0",
```

Comment:
Understood how this works with `npm ci`, but I think we might have to pin this in `package.json` - otherwise, a new developer who does a fresh `npm i` will retrieve the latest version

Reply:
just reverting so i can move forward with github advanced security. Once that is done i can pin it. lot of the new ones i have already started moving away from modifying lock and adding them in package as a resolution.

Reply:
ok, gotcha - I guess since we won't cut a new SDK version from this release, it should be fine /cc @kochis